### PR TITLE
Environment setting clarifications

### LINKED ISSUE #6605: Does IIS use ASPNETCORE_ENVIRONMENT?
The documentation is unclear on whether IIS uses ASPNETCORE_ENVIRONMENT or not.

At the opening of the Windows section, it describes how to set it for the current session for dotnet run. It then describes how to set it globally on Windows, without making clear if this still applies specifically to dotnet run.

After this, there is a section for web.config and IIS Application Pool. Both of these items address setting environment on individual apps. It is unclear if these sections are stating this in addition to being able to set it "globally on Windows".

I am currently unsuccessful in having applications hosted in IIS globally receive their environment from ASPNETCORE_ENVIRONMENT. Because of the ambiguity in the document, I am not sure if this is an error on my part or if it is not meant to work that way. 

I would like to be able to set the environment one time on a server and have every application in IIS use that environment. I would appreciate clarification on whether this is achievable or not.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ceffa6a2-770c-1cd9-5145-dcd423f9dd22
* Version Independent ID: 1cb5eb30-dd84-e5e1-5826-d74243fabf04
* Content: [Use multiple environments in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/environments?view=aspnetcore-2.0#setting-the-environment)
* Content Source: [aspnetcore/fundamentals/environments.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/fundamentals/environments.md)
* Service: **unspecified**
* Product: **asp.net-core**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #6605 

Guessing on ...

> Restart an app's app pool.

... as I haven't actually tried it myself. I'll test if needed.

### CODE REVIEW

File: aspnetcore/host-and-deploy/iis/index.md [hunk 1]
```diff
@@ -171,7 +171,7 @@ Enable the **IIS Management Console** and **World Wide Web Services**.
    
    To prevent the installer from installing x86 packages on an x64 OS, run the installer from an administrator command prompt with the switch `OPT_NO_X86=1`.
 
-1. Restart the system or execute **net stop was /y** followed by **net start w3svc** from a command prompt. Restarting IIS picks up a change to the system PATH made by the installer.
+1. Restart the system or execute **net stop was /y** followed by **net start w3svc** from a command prompt. Restarting IIS picks up a change to the system PATH, which is an environment variable, made by the installer.
```
Comment (Tratcher) [inline aspnetcore/host-and-deploy/iis/index.md]: @shirhatti is this better than IISReset?

Comment (guardrex) [reply aspnetcore/host-and-deploy/iis/index.md]: According to this, they stopped supporting it ... https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2008-R2-and-2008/dd364067(v=ws.10)

Comment (shirhatti) [reply aspnetcore/host-and-deploy/iis/index.md]: Yup. IISReset is no longer supported.

**Comment by guardrex** [comment]
@Tratcher Looking ok?

**Pull request merged**